lib/autoyast.pm

```diff
@@ -783,6 +784,48 @@ sub expand_agama_profile {
     return $profile_url;
 }
 
+=head2 install_jsonnet
```

Comment:
call it `workaround_*` and document it before some other contributors starts to scream :) because it is not good practice and we are being promised that will be packaged for all architectures but it wouldn't be the case we would need to rise the point or salt this other repo.